src/utils/init/utils.mjs

```diff
@@ -1,64 +1,72 @@
-
-const getDefaultBase = ({ baseDirectory, repositoryRoot }) => {
-  if (baseDirectory !== repositoryRoot && baseDirectory.startsWith(repositoryRoot)) {
-    return path.relative(repositoryRoot, baseDirectory)
-  }
-}
+// these plugins represent runtimes that are
+// expected to be "automatically" installed. Even though
+// they can be installed on package/toml, we always
+// want them installed in the site settings. When installed
+// there our build will automatically install the latest without
+// user management of the versioning.
+const pluginsToAlwaysInstall = new Set(['@netlify/plugin-nextjs'])
+
+/**
+ * Retrieve a list of plugins to auto install
+ * @param {string[]=} pluginsInstalled
+ * @param {string[]=} pluginsRecommended
+ * @returns
+ */
+export const getPluginsToAutoInstall = (pluginsInstalled = [], pluginsRecommended = []) =>
+  pluginsRecommended.reduce(
+    (acc, plugin) =>
+      pluginsInstalled.includes(plugin) && !pluginsToAlwaysInstall.has(plugin) ? acc : [...acc, plugin],
+
+    /** @type {string[]} */ ([]),
+  )
 
-const getDefaultSettings = ({
-  baseDirectory,
-  config,
-  frameworkBuildCommand,
-  frameworkBuildDir,
-  frameworkPlugins,
-  repositoryRoot,
-}) => {
-  const recommendedPlugins = getRecommendPlugins(frameworkPlugins, config)
-  const {
-    command: defaultBuildCmd = frameworkBuildCommand,
-    functions: defaultFunctionsDir,
-    publish: defaultBuildDir = frameworkBuildDir,
-  } = config.build
+/**
+ *
+ * @param {Partial<import('@netlify/build-info').Settings>} settings
+ * @param {*} config
+ * @param {import('../../commands/base-command.mjs').default} command
+ */
+const normalizeSettings = (settings, config, command) => {
+  const plugins = getPluginsToAutoInstall(settings.plugins_from_config_file, settings.plugins_recommended)
+  const recommendedPlugins = getRecommendPlugins(plugins, config)
 
   return {
-    defaultBaseDir: getDefaultBase({ repositoryRoot, baseDirectory }),
-    defaultBuildCmd,
-    defaultBuildDir: normalizeDir({ baseDirectory, dir: defaultBuildDir, defaultValue: '.' }),
-    defaultFunctionsDir: normalizeDir({ baseDirectory, dir: defaultFunctionsDir, defaultValue: 'netlify/functions' }),
+    defaultBaseDir: settings.baseDirectory ?? command.project.relativeBaseDirectory ?? '',
+    defaultBuildCmd: config.build.command || settings.buildCommand,
+    defaultBuildDir: settings.dist,
+    defaultFunctionsDir: config.build.functions || 'netlify/functions',
     recommendedPlugins,
   }
 }
 
+/**
+ *
+ * @param {object} param0
+ * @param {string} param0.defaultBaseDir
+ * @param {string} param0.defaultBuildCmd
+ * @param {string=} param0.defaultBuildDir
+ * @returns
+ */
 const getPromptInputs = ({ defaultBaseDir, defaultBuildCmd, defaultBuildDir }) => {
   const inputs = [
     defaultBaseDir !== undefined && {
       type: 'input',
       name: 'baseDir',
-      message: 'Base directory (e.g. projects/frontend):',
+      message: 'Base directory `(blank for current dir):',
```

Comment:
Either this or just "blank for current". Using "directory" and "dir" in the same sentence is not a good idea.

```suggestion
      message: 'Base directory `(blank for current directory):',
```